src/Test.UnitTests.Sarif.Driver/Sdk/SkimmerBaseTestResources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NotApplicable_InvalidMetadata" xml:space="preserve">
+    <value>'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}.</value>
```

Comment:
>as the analysis is not relevant [](start = 51, length = 31)

In fact, perhaps "as the analysis is not relevant" => "because the check does not apply to this target"? Up to you; I'm on a "simplify my English" kick.

Reply:
this is a copy of a string that's elsewhere in the SDK will scrub later.

---
In reply to: [322892738](https://github.com/microsoft/sarif-sdk/pull/1670#discussion_r322892738) [](ancestors = 322892738)